CMakeLists.txt

```diff
@@ -11,15 +12,34 @@ if(APPLE)
   endif()
 endif()
 
-# Name of the project
-# https://cmake.org/cmake/help/v3.0/command/project.html#command:project
 # Sets the env vars PROJECT_SOURCE_DIR, and PROJECT_BINARY_DIR
-project(zeth)
+project(zeth CXX)
 
 # Versionning of the project
 set(ZETH_VERSION_MAJOR 0)
 set(ZETH_VERSION_MINOR 3)
 
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+  # Support for the 2011 ISO C++ standard was claimed from gcc4.8+:
+  #     https://gcc.gnu.org/gcc-4.8/cxx0x_status.html
+  # And some sanitizers we use (e.g. UBSan) were added in gcc4.9+:
+  #     https://gcc.gnu.org/gcc-4.9/changes.html
+  if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 4.9)
+    message(FATAL_ERROR "We use c++11 and some sanitizers which require"
+      "GCC version at least 4.9")
```

Comment:
missign space in "requireGCC"

Reply:
Should be fixed in https://github.com/clearmatics/zeth/pull/208/commits/2be3993bf0895c2afc8b957158c31b94f61ff2e6